src/main/steps/children/nationality/content.ts

```diff
@@ -1,158 +1,54 @@
-import { TranslationFn } from '../../../app/controller/GetController';
-import { FormContent, FormFieldsFn } from '../../../app/form/Form';
-import { atLeastOneFieldIsChecked, isFieldFilledIn, notSure } from '../../../app/form/validation';
-import { mapSummaryListContent } from '../../common/functions/mapSummaryListContent';
+import { FieldPrefix } from '../../../app/case/case';
+import { PageContent, TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import {
+  nationalityFields,
+  form as nationalityForm,
+  generateContent as nationalityGenerateContent,
+} from '../../common/components/nationality';
 import { CHILDREN_NATIONALITY } from '../../urls';
 
-export const en = (): Record<string, unknown> => ({
+export const en = (nationalityContent: PageContent): Record<string, unknown> => ({
   section: "The child's details",
   label: 'What is their nationality?',
-  hint: 'Select all options that are relevant to you.',
-  british: 'British',
-  britishSubtext: 'including English, Scottish, Welsh and Northern Irish',
-  irish: 'Irish',
-  differentCountry: 'Citizen of a different country',
-  childrenNationality: 'Country name',
-  add: 'Add',
-  another: 'Add another country',
-  or: 'or',
-  notSure: 'Not Sure',
   errors: {
```

Comment:
I believe this is already included in the common component so error messages may be removed in the content.ts implementations